src/KoState.h

```diff
@@ -35,7 +35,7 @@ class KoState : public FastState {
     void play_move(int color, int vertex);
     void play_move(int vertex);
 
-private:
+//private:
```

Comment:
This kind of hacks will need to be fixed.

Reply:
I think making this single member public is harmless? I don't remember about other hacks like this.